website/docs/r/group_profile.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
+---
+
+# turbot_profile
+
+The `Turbot group profile` resource adds support for creating group profile under a directory type. It is used to create, manage and delete group profiles.
```

Comment:
I don't understand when I would use a group profile / create one in terraform. Please confirm with Tuhin or Mike for why / the use case.